composer.lock

```diff
@@ -8,7 +8,7 @@
     "packages": [
         {
             "name": "automattic/jetpack-asset-tools",
-            "version": "dev-master",
+            "version": "dev-update/package-tracks-move-more-things-into-package",
```

Comment:
Would be cool to revert all these branch-related changes and stick with the autoload-related ones.

Reply:
yes!

Reply:
Resolved in 56254490a99d8e2330aca8d7489fe8a01c011e2c